localstack/services/internal.py

```diff
@@ -144,7 +144,11 @@ def on_get(self, request):
         # using simple string replacement here, for simplicity (could be replaced with, e.g., jinja)
         for key, value in params.items():
             deploy_html = deploy_html.replace(f"<{key}>", value)
-        return deploy_html
+
+        response = Response()
+        response.data = deploy_html
+        response.headers["content-type"] = "text/html"
+        return response
```

Comment:
technically `"text/html"` is the mime-type, and `text/html; charset=utf-8` would be the `content-type`.
You can get the correct `content-type` header automatically by setting the `mimetype` attribute of the response.

This would IMO be the correct (and best way) to return this type of response:
```python
return Response(deploy_html, mimetype="text/html")
```

Reply:
Great catch! 👍 Updated